docs/source/userguide/mesh.rst

```diff
@@ -312,3 +312,296 @@ Meshes from FEniCS
 ------------------
 
 See the `FEniCS documentation <https://fenicsproject.org/olddocs/dolfin/latest/python/demos/built-in-meshes/demo_built-in-meshes.py.html>`_ for more built-in meshes.
+
+
+Meshes from GMSH
+----------------
+
+The Python API of GMSH can be used to create meshes that are useable in FESTIM. Here we will walk through its usage when creating a monoblock subsection consisting of tungsten surrounding a tube of CuCrZr
+
+.. thumbnail:: ../images/gmsh_tut_1.png
+    :width: 400
+    :align: center
+
+Meshing the geometry with GMSH
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Firstly, GMSH must be imported and initialised.
+
+.. code-block:: python
+    
+    import gmsh as gmsh
+    
+    gmsh.initialize()
+    gmsh.model.add("mesh")
+
+We can set the size of our mesh using:
+
+.. code-block:: python
+    
+    lc = 1e-3
+
+Models in GMSH consist of a series of:
+- Points
+- Lines
+-  Wires / Curve Loops
+   - whether we use curve loops or wires depends on whether we use the `.occ` or `.geo` geometry kernels. `.occ` allows for direct construction of more complex features such as cylinders, whereas using `.geo` requires explicit user definition of all the points, surfaces and volumes that would make up the cylinder. 
+-  Surfaces
+-  Surface Loops
+-  Volumes
```

Comment:
In the docs this doesn't seem to compile correctly, maybe it needs a blank line before the first bullet